modules/router/src/forwarding.ts

```diff
@@ -248,7 +281,7 @@ export async function forwardTransferCreation(
         amount: recipientAmount,
         assetId: recipientAssetId,
         routingId,
-        transferDefinition,
+        transferDefinition: senderTransferDefinition,
```

Comment:
`transferDefinition` here is incorrect, since it's referencing the incorrect definition (though correct `type`)

Reply:
Good catch. We aren't using this ATM but will fix now.